testing/h1-cli/APKBUILD

```diff
@@ -1,35 +1,37 @@
 # Contributor: Adam Dobrawy <a.dobrawy@hyperone.com>
 # Maintainer: Adam Dobrawy <a.dobrawy@hyperone.com>
+# Maintainer: HyperOne staff <pkg-maintainers@hyperone.com>
 pkgname=h1-cli
 _pkgname=${pkgname/-cli/}
-pkgver=1.6.0
+pkgver=1.6.1
 pkgrel=0
 pkgdesc="Command-line interface for HyperOne platform"
 url="https://www.hyperone.com/tools/cli/"
 arch="x86_64"
 license="MIT"
-depends="openssh-client"
+depends="openssh-client libstdc++"
```

Comment:
This change shouldn't be necessary `g++` depends on `libstdc++` and `build-base` depends on `g++`.